networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py

```diff
@@ -0,0 +1,228 @@
+import json
+import os
+
+import eventlet
+import netaddr
+from networking_nsxv3.common.constants import *
+from networking_nsxv3.plugins.ml2.drivers.nsxv3.agent import provider_nsx_mgmt
+from networking_nsxv3.plugins.ml2.drivers.nsxv3.agent.constants_nsx import *
+from oslo_config import cfg
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+
+class API(provider_nsx_mgmt.API):
+    POLICIES = "/policy/api/v1/infra/domains/default/security-policies"
+    POLICY = "/policy/api/v1/infra/domains/default/security-policies/{}"
+
+    GROUPS = "/policy/api/v1/infra/domains/default/groups"
+    GROUP = "/policy/api/v1/infra/domains/default/groups/{}"
+
+    STATUS = "/policy/api/v1/infra/realized-state/status"
+
+
+class Payload(provider_nsx_mgmt.Payload):
+
+    def get_compacted_cidrs(self, os_cidrs):
+        """
+        Reduce number of CIDRs based on the netmask overlapping
+        """
+        compacted_cidrs = []
+        for cidr in netaddr.IPSet(os_cidrs).iter_cidrs():
+            if cidr.version == 4 and cidr.prefixlen == 32:
+                compacted_cidrs.append(str(cidr.ip))
+            else:
```

Comment:
What about `cidr.version == 6 and cidr.prefixlen == 128`?

Reply:
- [x] Add IPv6 host CIDER normalization